src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5707,17 +5707,30 @@ namespace HeatBalanceSurfaceManager {
             }
 
             // fill in reporting values for outside face
-            QdotConvOutRep(SurfNum) = -Surface(SurfNum).Area * HcExtSurf(SurfNum) * (TH(1, 1, SurfNum) - Surface(SurfNum).OutDryBulbTemp);
+                        
+            QdotConvOutRepPerArea(SurfNum) = GetQdotConvOutRepPerArea(SurfNum);
+
+            QdotConvOutRep(SurfNum) = QdotConvOutRepPerArea(SurfNum) * Surface(SurfNum).Area;
 
-            if (Surface(SurfNum).OSCMPtr > 0) { // Optr is set above in this case, use OSCM boundary data
-                QdotConvOutRepPerArea(SurfNum) = -OSCM(OPtr).HConv * (TH(1, 1, SurfNum) - OSCM(OPtr).TConv);
-            } else {
-                QdotConvOutRepPerArea(SurfNum) = -HcExtSurf(SurfNum) * (TH(1, 1, SurfNum) - Surface(SurfNum).OutDryBulbTemp);
-            }
             QConvOutReport(SurfNum) = QdotConvOutRep(SurfNum) * TimeStepZoneSec;
+
         } // ...end of DO loop over all surface (actually heat transfer surfaces)
     }
 
+    Real64 GetQdotConvOutRepPerArea(int const SurfNum) 
+    {
+        int OPtr = Surface(SurfNum).OSCPtr;
```

Comment:
@dareumnam Some integration tests are failing. Here's the problem - this should be `.OSCMPtr`.

Reply:
@mjwitte Oops, thank you for letting me know. I'll fix it and push the branch again.